codesign.py

```diff
@@ -151,18 +153,15 @@
                     ],
                 },
             'Flutter.xcframework/ios-arm64_x86_64-simulator/Flutter.framework/Flutter',
-            'Flutter.xcframework/ios-arm64_armv7/Flutter.framework/Flutter',
```

Comment:
This wasn't removed, it was renamed to `Flutter.xcframework/ios-arm64/Flutter.framework/Flutter`.

Reply:
@XilaiZhang is this what we're running in production?

Reply:
in any case, we should fix this repo, and then pull it onto the codesigning bot

Reply:
Thanks for the updates! Umm that’s interesting, I wonder if we have missed this artifact in release the entire time

Reply:
Sorry for the potential delay but I will take a look later. I headed out when Christopher started online chess

Reply:
Thanks for pointing this out! added paths for ios, ios-release and ios-profile, PR at https://github.com/christopherfujino/codesign.py/pull/4

Reply:
> I wonder if we have missed this artifact in release the entire time

It was renamed from `ios-arm64_armv7` to `ios-arm64`.  See https://github.com/flutter/flutter/pull/102012.